docs/wpf/getting-started.md

```diff
@@ -1 +1,299 @@
+
+## Install the Nuget Packages
+
+Create a brand new WPF application in Visual Studio. Next up is to install the appropriate nuget packages. At this point, a choice needs to be made, and that is which container to use for managing dependencies. For the purposes of this documentation, Unity will be the container of choice. See the list below of what is available.
+
+| Package | Container | Version |
+|---------|-----------|---------|
+| Prism.Unity   | [Unity](https://github.com/unitycontainer/unity) | 5.11.1 |
+| Prism.Dryloc  | [DryLoc](https://github.com/dadhi/DryIoc)        | 4.0.7  |
+| Prism.Ninject | [Ninject](http://www.ninject.org)                | 3.3.4  |
+
+> Note: There is no need to explicitly install any other dependencies. Installing one of the above packages will also take care of installing the packages for the container as well as the shared Prism packages.
+
+![Install Nuget](images/nuget-install.png)
+
+## Override the Existing Application Object
+
+The next step in getting started is to subclass the Application object contained in the newly created WPF project. Navigate to the ```App.xaml``` and  replace the standard WPF Application class with the Prism ```PrismApplication``` class.
+
+```xml
+<prism:PrismApplication
+    x:Class="WpfApp1.App"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="clr-namespace:WpfApp1"
+    xmlns:prism="http://prismlibrary.com/">
+    <Application.Resources>
+    </Application.Resources>
+</prism:PrismApplication>
+```
+
+In the snippet above, notice that line 6 has been added to define the namespace and that the App object has been updated to derive from ```PrismApplication```. Next, navigate to the ```app.xaml.cs``` code-behind file and update the class definition.
+
+> Don't forget to remove the ```StartupUri``` property from the ```PrismApplication``` tag. Otherwise, you will end up with two window instances.
+
+
+```cs
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+using Prism.Unity;
+
+namespace WpfApp1
+{
+    public partial class App : PrismApplication
+    {
+    }
+}
+```
+
+There are a pair of abstract methods defined in ```PrismApplication``` that must be implemented first: RegisterTypes and CreateShell.
+
+### RegisterTypes
+
+This function is used to register any app dependencies. For example, there might be an interface to read customer data from a persistent store of some kind and the implementation of it is to use a database of some kind. It might look something like this:
+
+```cs
+public interface ICustomerStore
+{
+    List<string> GetAll();
+}
+
+public class DbCustomerStore()
```

Comment:
class definition is wrong here and does not inherit the interface

Reply:
Fixed.